core-ui/src/shared/components/ResourceRef/ExternalResourceRef.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FormInput, ComboboxInput } from 'fundamental-react';
+
+import './ExternalResourceRef.scss';
+
+export function ExternalResourceRef({ resourceRef, onChange, resources }) {
+  const { t } = useTranslation();
+  const options = (resources || []).map((resource, index) => ({
+    key: index,
+    text: resource.metadata.name,
+    namespace: resource.metadata.namespace,
+  }));
+
+  const selectResource = (e, selected) => {
+    if (selected.key > -1) {
+      onChange(e, { name: selected.text, namespace: selected.namespace });
+    }
+  };
+
+  return (
+    <div className="external-resource-ref">
+      <ComboboxInput
+        compact
+        onChange={e => {
+          onChange(e, { ...resourceRef, name: e.target.value });
+        }}
+        onSelectionChange={selectResource}
+        placeholder={t('common.labels.name')}
+        formItemProps={{ className: 'name' }}
+        inputProps={{ value: resourceRef.name }}
+        options={options}
+        optionRenderer={resource => (
+          <>
+            <span class="fd-list__title">{resource.text}</span>
+            <span className="fd-list__secondary">{resource.namespace}</span>
+          </>
+        )}
+      />
+      <div className="namespace fd-input-group--control fd-input-group">
+        <FormInput
```

Comment:
The idea was to allow both name and namespace to be entered manually as well, not only via searching

Reply:
sure, but if I write a namespace, then I would expect the secrets list to be filtered by this namespace.
it's not a critical issue since it's not a bug, just 2 different points of view. 🙂